_data/toc.yaml

```diff
@@ -1030,6 +1030,8 @@ reference:
         title: v1.18 reference
   - title: Docker Hub API
     path: /docker-hub/api/latest/
+  - title: DVP Data API
+    path: /docker-hub/api/dvp/
```

Comment:
I wonder if we should start to consider making `Docker HUB API` a sub-section under which we'll list the Docker Hub related APIs. I know we discussed that some time ago, but at the time we only had a single API, so no grouping was needed, but now that there's two (or three, if we would consider the "registry API" to be part), perhaps it makes sense to start doing so.

This would also allow for an `index.md` / overview to be added that outlines what APIs exist (and for which purpose), and possibly other content to introduce users to the APIs (and where to learn more), or highlight "changes" (if relevant), similar to the "Engine API" section.

<img width="277" alt="Screenshot 2022-07-04 at 11 09 20" src="https://user-images.githubusercontent.com/1804568/177123277-18e281eb-f99a-4d65-9f34-80fd42d0de0e.png">

Reply:
Could do. Not sure who would be best placed to own that content though...

Reply:
Additional content/markdown (if we choose to add it) would be maintained in this repository, but can be added later. Changing the TOC to use a section would look something like;

```yaml
  - sectiontitle: Docker Hub API
    section:
      - title: Docker Hub API
        path: /docker-hub/api/latest/
      - title: DVP Data API
        path: /docker-hub/api/dvp/
  - title: Registry API
    path: /registry/spec/api/
  - sectiontitle: Docker Extension SDK API (Beta)
  # ...
```

Reply:
Sure, I meant more of whose responsibility would that be?

Reply:
And do we want to tackle that now or add it to the backlog for the next iteration?

Reply:
If we consider it a good change to have the Docker Hub API's grouped in the navigation, I would consider doing that as  part of this PR (as this changes the docs to have multiple docs for the hub API).

w.r.t "responsibility" (assuming you're talking about the other content (index page, change log, etc); I think it'll be difficult to make a "hard cut" between who's responsible; I'd consider if a shared responsibility for all teams working on Docker Hub APIs. I would expect maintenance for an overview page to be low (changes would be needed only if a new API is added, e.g., but of course depends on what we want to show on it). Perhaps someone from the documentation team could write a first draft for an index page.

Reply:
OK, we'll I've grouped it for now but haven't added an index page or anything.